lib/index.js

```diff
@@ -327,31 +349,31 @@ King = new Class({
         url = this.url( 'apis', id, 'plugins');
         debug('request url %s', url, data );
 
-        plugins_list = yield request({
+        plugins_list = yield this.request({
             method:'GET'
           , uri:url
           ,json:true
-        });
+        }, [200]);
 
-        plugins = plugins_list.body.data;
+        plugins = plugins_list.body.data || [];
```

Comment:
This patch preceded the fix with errors. This is how I started unraveling that errors weren't being handled at all.